client/my-sites/plans/main.jsx

```diff
@@ -36,9 +41,54 @@ const Plans = React.createClass( {
 		};
 	},
 
+	shouldDisplayJetpackPlansDisabled() {
+		const { selectedSite } = this.props;
+
+		return selectedSite && selectedSite.jetpack && ! isEnabled( 'manage/jetpack-plans' );
+	},
+
+	componentDidMount() {
+		const { context, selectedSite } = this.props,
+			analyticsPageTitle = 'Plans',
+			basePath = route.sectionify( context.path ),
+			analyticsBasePath = ( selectedSite ) ? basePath + '/:site' : basePath;
+
+		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		context.store.dispatch( setTitle( i18n.translate( 'Plans', { textOnly: true } ) ) );
```

Comment:
I would suggest using `<DocumentHead>` instead according to the FIXME comment ;)
You just need to put it in the `render` method in a similar way we do it with query components.